modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -0,0 +1,219 @@
+= Guide to Named Examples
+
+If you want to use more than one example for a type, resource type, or trait, you must name the examples. You can also name a single example. When you name examples, you must use the `examples` facet.
+
+To use one example without naming it, you must use the `example` facet.
+
+
+== Naming a single example in the `examples` facet
+
+In the sample files in this section, the `examples` facet names the example `fullName`. The fragment is not declared as a NamedExample fragment because it does not provide a name.
```

Comment:
"In the sample files in this section,..."
> "In the following sample files,..."

Should the narrative refer to ".api.raml" vs. ".fragment.raml," for comparison and clarity?